packages/cpp/ArmoniK.Api.Worker/header/Worker/TaskHandler.h

```diff
@@ -89,7 +84,8 @@ class TaskHandler {
    * @param data The result data
    * @return A future containing a vector of ResultReply
    */
-  std::future<armonik::api::grpc::v1::agent::ResultReply> send_result(std::string key, absl::string_view data);
+  std::future<armonik::api::grpc::v1::agent::NotifyResultDataResponse> send_result(std::string key,
```

Comment:
It is unfortunate that it changes the API. I think you should wrap this message somehow

Reply:
I'd return nothing and throw if there is an issue